### PR TITLE
Feature/reliability service

### PR DESCRIPTION
**Pull request by ctacke**

### CODE REVIEW

File: source/Meadow.Core/Hardware/BiDirectionalInterruptPort.cs [hunk 1]
```diff
@@ -130,6 +135,7 @@ OutputType outputType
                 if (_currentDirection == PortDirectionType.Input)
                 {
                     // This call will ultimately result in Nuttx being called
+                    Resolver.Log.Info($"configure for interrupt: {interruptMode}");
```
Comment (adrianstevens) [inline source/Meadow.Core/Hardware/BiDirectionalInterruptPort.cs:138]: should we delete the Log.Info call?

File: source/Meadow.Core/Hardware/BiDirectionalInterruptPort.cs [hunk 2]
```diff
@@ -234,7 +254,9 @@ public override bool State
             get
             {
                 Direction = PortDirectionType.Input;
+                Resolver.Log.Info($"Reading state...");
```
Comment (adrianstevens) [inline source/Meadow.Core/Hardware/BiDirectionalInterruptPort.cs:257]: And here

File: source/Meadow.Core/Hardware/BiDirectionalInterruptPort.cs [hunk 3]
```diff
@@ -272,6 +294,7 @@ public override TimeSpan GlitchDuration
 
         private void OnInterrupt(IPin pin, bool state)
         {
+            Resolver.Log.Info($"interrupt!");
```
Comment (adrianstevens) [inline source/Meadow.Core/Hardware/BiDirectionalInterruptPort.cs:297]: And here

**Review by adrianstevens — APPROVED** [review]

File: source/Meadow.Core/Hardware/BiDirectionalInterruptPort.cs (hunk 1 above)
Comment (jorgedevs) [reply source/Meadow.Core/Hardware/BiDirectionalInterruptPort.cs:138]: Agree, or set it to Trace

**Pull request merged**